Presentation.Web/Models/API/V2/Internal/Response/ItSystemUsage/ItSystemUsageRelationMigrationV2ResponseDTO.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Presentation.Web.Models.API.V2.Response.Generic.Identity;
+
+namespace Presentation.Web.Models.API.V2.Internal.Response.ItSystemUsage
+{
+    public class ItSystemUsageRelationMigrationV2ResponseDTO
+    {
+        public IdentityNamePairWithDeactivatedStatusDTO ToSystemUsage { get; set; }
+        public IdentityNamePairWithDeactivatedStatusDTO FromSystemUsage { get; set; }
+        public string Description { get; set; }
+        public IdentityNamePairResponseDTO Interface { get; set; }
```

Comment:
are all of the properties here needed for the ui use case? If so, then it's fine